apps/web/src/containers/AnalyticsView/hooks/use-formatted-data.tsx

```diff
@@ -0,0 +1,68 @@
+import { Network, getEtherscanTransactionLink } from '@notional-finance/util';
+import { ReceivedIcon, SentIcon } from '@notional-finance/icons';
+import { formatTokenAmount, formatTokenType } from '@notional-finance/helpers';
+import { useMemo } from 'react';
+import { useTheme } from '@mui/material';
+
+export const useFormattedData = (data: any[], network?: Network) => {
```

Comment:
Can this be reused in `use-txn-history-data`? Prefer not to have duplicate logic